src/module/iotjs_module_fs.cpp

```diff
@@ -139,8 +138,16 @@ JHANDLER_FUNCTION(Open) {
   if (handler.GetArgLength() > 3 && handler.GetArg(3)->IsFunction()) {
     FS_ASYNC(env, open, handler.GetArg(3), path.data(), flags, mode);
   } else {
+    /* FIXME it is workaround of valgrind error on NULL value as 'stat' input */
+    /* Empty string should point some valid empty string */
+    char tmp[1] = { '0' };
```

Comment:
FYI, in order to change C++ to C, these are transient workaround codes to enable valgrind.

`\0` is true NULL string. As I write above at FIXME, valgrind shows error at NULL value as unaddressable byte on `stat` input.
Typical error messages of valgrind look like:
```
==26986== Syscall param stat(file_name) points to unaddressable byte(s)
==26986==    at 0x5758045: _xstat (xstat.c:35)
==26986==    by 0x4F16F1: uv__fs_stat (uv_unix_fs.c:188)
==26986==    by 0x4F2236: uv__fs_work (uv_unix_fs.c:483)
==26986==    by 0x4F29C4: uv_fs_stat (uv_unix_fs.c:589)
==26986==    by 0x41D741: iotjs::___Stat_native(iotjs::JHandlerInfo&) (iotjs_module_fs.cpp:304)
==26986==    by 0x41D3CE: iotjs::Stat(unsigned int, unsigned int, unsigned int const*, unsigned int) (iotjs_module_fs.cpp:287)
==26986==    by 0x4C6F0E: jerry_dispatch_external_function (jerry.c:2391)
==26986==    by 0x4B1EE1: ecma_op_function_call (ecma-function-object.c:587)
==26986==    by 0x4E4993: opfunc_call (vm.c:380)
==26986==    by 0x4EC66B: vm_execute (vm.c:2599)
==26986==    by 0x4EC8E6: vm_run (vm.c:2669)
==26986==    by 0x4B1E6C: ecma_op_function_call (ecma-function-object.c:568)
==26986==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
```